pkg/rpcclient/actor/actor.go

```diff
@@ -25,6 +26,7 @@ import (
 // create and send transactions.
 type RPCActor interface {
 	invoker.RPCInvoke
+	SimpleWaiter
```

Comment:
Also, I'm thinking about moving `SimpleWaiter` from `RPCActor` to `invoker.RPCInvoke`, because the user may need to await transaction without its creation and Actor usage.

Reply:
Not a likely scenario, I think.